### PR TITLE
Variable usage and cleanup

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

File: dist/css/rise.css [hunk 1]
```diff
@@ -8076,23 +8076,6 @@ All utilities have a prefix u_ added to them
...
-    /* TODO: Make this a variable. */ }
-  .stripe {
-    margin-bottom: 0; } }
-
-/* ==========================================================================
-   Print Media Queries
-   ========================================================================== */
-@media print {
-  /* Remove header when printing. */
-  .navbar {
```
Comment (RV-Adrian) [inline dist/css/rise.css]: Used 'hidden-print' class instead.

File: dist/css/rise.css [hunk 2]
```diff
@@ -8076,23 +8076,6 @@ All utilities have a prefix u_ added to them
   height: 24px;
   background: url("//s3.amazonaws.com/rise-common-test/images/glyphicons-back.png") center no-repeat; }
 
-/* Media queries
-   ========================================================================== */
```
Comment (RV-Adrian) [inline dist/css/rise.css]: Did not find usage on these.

File: dist/css/rise.css [hunk 3]
```diff
@@ -8076,23 +8076,6 @@ All utilities have a prefix u_ added to them
...
-  .modal-dialog .sign-in {
-    padding: 3px 10px;
-    /* TODO: Make this a variable. */ }
-  .stripe {
-    margin-bottom: 0; } }
-
-/* ==========================================================================
-   Print Media Queries
-   ========================================================================== */
-@media print {
```
Comment (RV-Adrian) [inline dist/css/rise.css]: Used 'hidden-print' class instead.

**Comment by RV-Adrian** [comment]
@ghislaineguerin 
Please review for https://trello.com/c/JuQ421YQ/1944-css-responsive-utilities-global-vs-container-specific-3

Mostly using variable names instead of pixels, and cleaning up unused classes.
Some specific inline notes are above. Thanks!

**Comment by ghislaineguerin** [comment]
@RV-Adrian 👍

**Comment by RV-Adrian** [comment]
@ghislaineguerin Round two! Made some changes as per our conversation re: ellipsis usage.
All classes of '.ellipsed' are now removed, and use new utilities. 
See staging at: https://apps-stage-3.risevision.com
Thanks!

**Comment by ghislaineguerin** [comment]
👍 looks great

**Pull request merged**